xmppserver/src/main/java/org/jivesoftware/openfire/disco/IQDiscoInfoHandler.java

```diff
@@ -191,12 +194,14 @@ public IQ handleIQ(IQ packet) {
                     // discovery broadcasts the disco#info feature.
                     queryElement.addElement("feature").addAttribute("var", NAMESPACE_DISCO_INFO);
                 }
-
-                // Add to the reply the extended info (XDataForm) provided by the DiscoInfoProvider
-                DataForm dataForm = infoProvider.getExtendedInfo(name, node, packet.getFrom());
-                if (dataForm != null) {
+                // Add to the reply the multiple extended info (XDataForm) provided by the DiscoInfoProvider
+                Iterator<DataForm> dataForms = infoProvider.getExtendedInfos(name, node, packet.getFrom()).iterator();
+                //Log.info("DATAFORM {}",dataForms.toString());
```

Comment:
I think you used this Log statement for debugging purposes. If we're not going to use it, let's remove it completely.